tests/utils/test_rest_utils.py

```diff
@@ -65,97 +65,106 @@ def test_malformed_json_error_response(response_mock):
             call_endpoint(host_only, "/my/endpoint", "GET", "", response_proto)
 
 
-@mock.patch("requests.request")
+@mock.patch("requests.Session.request")
 def test_http_request_hostonly(request):
     host_only = MlflowHostCreds("http://my-host")
     response = mock.MagicMock()
     response.status_code = 200
     request.return_value = response
-    http_request(host_only, "/my/endpoint")
+    http_request(host_only, "/my/endpoint", method="GET")
```

Comment:
http_request() function needs method as an arg. This is the case before the change as well. I'm considering making it mandatory. Let me know what you think.

Reply:
Sounds great! Let's do it. `method` is required by `requests.request`, so we should require it explicitly in the signature.

Reply:
+1 for making `method` mandatory.

Reply:
Done.